2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex

```diff
@@ -64,11 +64,11 @@
 \end{itemize}
 
   \begin{block}{Why sensitivity analyses?}
-  The main objectives of sensitivity analyses are the following:
+  The main objectives of sensitivity analyses may be:
 \begin{enumerate}
  \item \alert{remove some variables} which are not influential on the feature of interest, within a context of high dimension,
- \item\alert{ prioritize variables } in order to prioritize modeling efforts: we need a  \emph{relative} quantification: we want that if $S_i < S_j$ then $\Prob{\hat{S}_j \leq \hat{S}_i} \leq \varepsilon$
- \item \alert{quantify the impact of a variable}: we need a \emph{exact} quantification: we want that $\Prob{|\hat{S}_i - S_i| > \eta} \leq \varepsilon$
+ \item\alert{ prioritize variables } in order to prioritize modeling efforts: we need a  \emph{relative} quantification
```

Comment:
I would say "Rank variables n order to priotirize.."

Reply:
Fixed, and rephrased.